x/evm/types/params.go

```diff
@@ -23,6 +24,9 @@ var (
 	DefaultEnableCreate = true
 	// DefaultEnableCall enables contract calls (i.e true)
 	DefaultEnableCall = true
+	// DefaultActivePrecompiles defines the default active precompiles
+	// TODO: testing if this fixes the issues with integration tests
```

Comment:
@GAtom22 Has this TODO been resolved?

Reply:
Yeah, I'll remove this TODO. @fedekunze Should we include in this array the Bech32 precompile too?

Reply:
no, let's add it later